include/marl/event.h

```diff
@@ -0,0 +1,130 @@
+// Copyright 2019 The Marl Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef marl_event_h
+#define marl_event_h
+
+#include "conditionvariable.h"
+#include "memory.h"
```

Comment:
maybe unrelated to this patch, but shouldn't the includes be "marl/conditionvariable.h" and "marl/memory.h" instead. Otherwise this may include the wrong header depending on how the include search path list was setup at compile time.

Reply:
humm, I had assumed that includes with no directory prefix would search relative directory first, but you're probably right.
I'll create a bug to address this across the whole project.

Reply:
https://github.com/google/marl/issues/54